src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -1906,6 +1906,14 @@ Every bucket is identified by a unique bucket name. The bucket naming should mat
 
 * A bucket name must start with a lowercase letter and cannot terminate with a dash.
 
+=== Object naming requirements
+
+Each Catalog object is identified by a unique object name. The object naming should match the following rules:
```

Comment:
`Each Catalog object is identified by a unique object name.` => `Each Catalog object is identified by a unique name.` (to prevent redundancy)

Reply:
I would also suggest to say:
The object naming should respect the following rules:
instead of match because you're talking about the act of naming objects and not about the name itself